postgresql/default.toml

```diff
@@ -11,6 +12,8 @@ password = 'admin'
 [replication]
 name = 'replication'
 password = 'replication'
+synchronous = true
+synchronous_standby_names = '*'
```

Comment:
I think we have a lot of customers who would want this - but to make it useful you actually need to manage the `application_name` of the standby servers. Probably too complex for the initial release. 

I'll take it out and always use * and maybe return to it later.